test/ToneAnalyzer.v3.UnitTests/ToneAnalyzerServiceUnitTests.cs

```diff
@@ -1,410 +0,0 @@
-﻿/**
-* Copyright 2017 IBM Corp. All Rights Reserved.
-*
-* Licensed under the Apache License, Version 2.0 (the "License");
-* you may not use this file except in compliance with the License.
-* You may obtain a copy of the License at
-*
-*      http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
-*
-*/
-
-using IBM.Cloud.SDK.Core.Authentication.NoAuth;
-using IBM.Cloud.SDK.Core.Http;
-using IBM.Cloud.SDK.Core.Http.Exceptions;
-using IBM.Watson.ToneAnalyzer.v3.Model;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using NSubstitute;
-using System;
-using System.Collections.Generic;
-using System.Net;
-using System.Net.Http;
-using System.Threading.Tasks;
-
-
-namespace IBM.Watson.ToneAnalyzer.v3.UnitTests

```

Comment:
Did you mean to delete this? I didn't see any of the old handwritten unit test files get deleted.

Reply:
Yeah, this one was problematic and commented out anyway (line 33).